.gitignore

```diff
@@ -2,6 +2,7 @@ build
 site/
 
 ui/elm-stuff
+ui/index.html
```

Comment:
Why are you ignoring `index.html`?

Reply:
Because it is auto-generated by `elm make` from elm code. Elm islike a react - all HTML code is inside elm components.